### DISCUSSION TITLE
Downgraded to a personal account and lost access to a domain.

### DISCUSSION THREAD

**Discussion by dave-hawkins** [vercel/community#5973] [Account Issue] [answered]
### Question

I had a domain purchased through a team account and deleted the team without realising the domain was there. I've now completely lost access to the domain. How do I get it back? Can someone transfer it to my existing account?

**Comment by younes0** [comment]
@dave-hawkins not sure it's related but since the update which moves your personal projects to your "personal team", I wasn't finding my domains. I had to go to https://vercel.com/account/domains find them and move them to my "personal team"

**Reply by dave-hawkins** [reply, marked as answer]
@younes0 Thanks - I reached out to vercel support via email and they helped me with this in the end. 

I'll put their instructions here for others:

I've had a look at the domain and I can see it's not currently owned by any account and instead is just added to your project.
I believe the team deletion warning advises to move the domains, but this is often missed resulting in the domain being orphaned. 

Since the domain isn't owned by any account, you should be able to add this to your personal account by going to the Domains page, then pressing the "Add" button and following the steps to add the domain.

Given the apex domain ([******.dev](http://******.dev/)) is already assigned to your project, it's likely you'll need to add a subdomain such as "[****.******.dev](http://****.******.dev/)". Once this has been added, the domain ownership will be added to your account and you can then remove the temp domain from your project settings (not the domains page).

**Reply by younes0** [reply]
@dave-hawkins thanks, I didn't have this issue but this is very infromative.
